test/system/710-start.bats

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bats   -*- bats -*-
+
+load helpers
+
+function setup() {
+    basic_setup
+    run_podman run -d $IMAGE /bin/true
+    cid_none="$output"
+    run_podman run -d --restart=always $IMAGE /bin/true
+    cid_always="$output"
+}
+
+function teardown() {
+    basic_teardown
+}
+
+@test "podman start --all - start all containers" {
+    run_podman start --all
+    is "$output" "$cid_none
+$cid_always"
+}
```

Comment:
It would probably be good to have a failure case or two here.  i.e.  `--all` and `--latest` in one line and/or `--all` and  a containerID or two.

Reply:
Right, I forgot about that. Thanks for mentioning it.